charts/nri-metadata-injection/templates/deployment.yaml

```diff
@@ -9,7 +9,9 @@ spec:
   replicas: {{ .Values.replicas }}
   selector:
     matchLabels:
-      {{- include "newrelic.common.labels.selectorLabels" . | nindent 6 }}
+      {{- /* We cannot use the common library here because of a legacy issue */}}
+      {{- /* `selector` is mnmutable and the previous chart did not have all the idiomatic labels */}}
```

Comment:
```suggestion
      {{- /* `selector` is immutable  and the previous chart did not have all the idiomatic labels */}}
```

Reply:
I wanted to fix a typo and introduced two 😅

Thanks!